docs/EN_US/HPCCSpark/SparkHPCC.xml

```diff
@@ -276,19 +276,19 @@
                   </row>
 
                   <row>
-                    <entry>SPARK_MASTER_WEBUI_PORT</entry>
+                    <entry>SPARK_MANAGER_WEBUI_PORT</entry>
 
-                    <entry>Base port to use for master web interface</entry>
+                    <entry>Base port to use for manager web interface</entry>
 
                     <entry>8080</entry>
 
                     <entry>optional</entry>
                   </row>
 
                   <row>
-                    <entry>SPARK_MASTER_PORT</entry>
+                    <entry>SPARK_MANAGER_PORT</entry>
```

Comment:
@Michael-Gardner please confirm this variable name change is valid

Reply:
Again this one needs to be reverted for the same reason, to SPARK_MASTER_PORT